apps/wallet/src/ui/app/components/accounts/AccountListItem.tsx

```diff
@@ -4,16 +4,18 @@
 import { useResolveSuiNSName } from '@mysten/core';
 import { formatAddress } from '@mysten/sui.js/utils';
 
-import { type ReactNode, useState } from 'react';
+import { type ReactNode } from 'react';
 import { AccountItem } from './AccountItem';
 import { LockUnlockButton } from './LockUnlockButton';
 import { useActiveAddress } from '../../hooks';
+import { useAccounts } from '../../hooks/useAccounts';
 
 type AccountListItemProps = {
 	address: string;
```

Comment:
ended up updating this to take in `account`, since we were looking up the account anyway from the `useAccounts` list in order to get props like `isLocked` `nickname` etc.